webextensions/api/management.json

```diff
@@ -363,10 +369,12 @@
                 "version_added": "79"
               },
               "firefox": {
-                "version_added": false
+                "version_added": "55",
+                "notes": "Before version 56, only extensions whose 'type' is 'theme' are supported. Also note that this event is not emitted when the extension is in the \"pending uninstall\" state: once it is completely removed (e.g. when the `about:addons` tab is closed or refreshed) the event is emitted as expected."
```

Comment:
suggestion: BCD accepts an array of notes, when the notes refer to different issues. So I'd recommend breaking this up into something like this:



```
[
  "Before version 56, only extensions whose <code>type</code> is <code>'theme'</code> are supported".,
  "This event is not emitted when the extension is in the \"pending uninstall\" state. The event is emitted as expected once the extension is completely removed (for example, when the <code>about:addons</code> tab is closed)."
]
```

(OK, I also did some wordsmithing and markup changes. I'm a technical writer and I cannot help myself. 😄)

This kind of change would need to be applied to the other below note as well.

Reply:
> 
> 
> suggestion: BCD accepts an array of notes, when the notes refer to different issues. 

TIL!

> (OK, I also did some wordsmithing and markup changes. I'm a technical writer and I cannot help myself. 😄)
> 
> This kind of change would need to be applied to the other below note as well.

This is great! Thanks for this, I'm pushing an update now :)